lib/ui/common/stage_card.dart

```diff
@@ -0,0 +1,184 @@
+import 'package:farmsmart_flutter/ui/common/Dogtag.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+export 'package:farmsmart_flutter/ui/common/Dogtag.dart';
+export 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+
+class StageCardViewModel {
+  String stageNumber;
+  String stageTitle;
+  String actionButtonText;
+  String stageStatusTitle;
```

Comment:
I think "statusTitle" as the widget is called stage card so we don't need the stage bit